app/scripts/views/sign_up_password.js

```diff
@@ -17,6 +17,16 @@ import ServiceMixin from './mixins/service-mixin';
 import SignUpMixin from './mixins/signup-mixin';
 import Template from 'templates/sign_up_password.mustache';
 
+function selectAutoFocusEl(password, vPassword) {
```

Comment:
I didn't even think to add it here! Well spotted!

Reply:
Thanks!